test/MissionTest.t.sol

```diff
@@ -96,18 +96,15 @@ contract MissionTest is Test {
         vm.stopPrank();
     }
 
-    function testCreateTeamProject() public {
+    function testCreateTeamProject1() public {
```

Comment:
Curious what would happen if someone sets their goal to 0

Reply:
added that unit test. it would proceed straight to open funding.